common/src/lib.rs

```diff
@@ -44,3 +47,33 @@ pub fn create_custom_connection<Client: FromStr + RpcClient>(
         }
     }
 }
+
```

Comment:
I'm confused by these changes – am I right in thinking they are only used by `fork-off`? If so, maybe put them in that crate rather than here? And even then, some of them seem to be unused?

I might be lacking some context or GitHub is lying to me with what it is displaying though. <_<"

Reply:
fixed - moved the fns to main module